PowerEditor/src/ScintillaComponent/FindReplaceDlg.cpp

```diff
@@ -3988,11 +3988,9 @@ void Finder::removeAll()
 
 void Finder::openAll()
 {
-	size_t sz = _pMainFoundInfos->size();
-
-	for (size_t i = 0; i < sz; ++i)
+	for (generic_string path : getResultFilePaths())

```

Comment:
It's better to declare `path` as `const generic_string&` or `generic_string&&` or even `auto&&` to avoid unnecessary copies.

Reply:
Sure, why not? :-)